CodeMaid.config

```diff
@@ -8,7 +8,7 @@
   <userSettings>
     <SteveCadwallader.CodeMaid.Properties.Settings>
       <setting name="Cleaning_AutoCleanupOnFileSave" serializeAs="String">
-        <value>True</value>
+        <value>False</value>
```

Comment:
Turned off CodeMaid's auto cleanup on save in order to avoid this bug https://github.com/codecadwallader/codemaid/issues/860